src/plugins/grpc/server.js

```diff
@@ -0,0 +1,137 @@
+'use strict'
+
+const { FORMAT_HTTP_HEADERS, Tags } = require('opentracing')
+
+let patchedHandlers = []
+
+function createWrapHandler (grpc, tracer, config, handler) {
+  return function wrapHandler (func) {
+    return function funcHandler (emitter) {
```

Comment:
Preserve the original function name with `withTrace` suffix for stack traces.

Reply:
Gotcha :+1: